authors/apps/articles/models.py

```diff
@@ -79,4 +79,7 @@ def clear_tags(self):
         """
         Clears all tags for an article
         """
-        [self.tags.remove(tag) for tag in self.tags.all()]
```

Comment:
Hello @oluwagbenga-joloko ,
I thought of it like any other custom property field of the model. However, since this is not actually the case, I think making it a method would be a better implementation. I have update it.